app.js

```diff
@@ -0,0 +1,32 @@
+const express = require ("express")
+const {getTopics} = require("./controllers/topics.controllers")
+
+const app = express ()
+
+app.get("/api/topics", getTopics)
+
+//error handling middleware starts here
+app.use((err, req, res, next) => {
+    console.log(err,"<--full error")
+
+    if (err.code) {
+      res.status(400).send({ msg: "Bad request" });
+    } else {
+      next(err);
+    }
```

Comment:
Are we expecting to hit a 400 with this endpoint? Presumably not, so why is this here? 👀